runtime.md

```diff
@@ -132,16 +132,26 @@ This operation MUST generate an error if `process` was not set.
 `kill <container-id> <signal>`
 
 This operation MUST [generate an error](#errors) if it is not provided the container ID.
+
 Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
-This operation MUST send the specified signal to the container process.
+
+This operation MUST send the specified signal to the container's init process.
+
+Specially, if the signal is `SIGKILL` and the container does not use its own private PID namespace, this operation MUST send the `SIGKILL` signal to all the processes in the container, even if the container's state is `stopped`. If there is no process left in this type container, the operation MUST [generate an error](#errors).
 
 ### <a name="runtimeDelete" />Delete
 `delete <container-id>`
 
 This operation MUST [generate an error](#errors) if it is not provided the container ID.
+
 Attempting to `delete` a container that is not [`stopped`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
+
 Deleting a container MUST delete the resources that were created during the `create` step.
+
+Specially, when deleting a container, which does not use its own private PID namespace, the operation should ensure kill all processes in this type container, and ensure no process left in it.
```

Comment:
`s/does not use/does not have/`

`s/should ensure kill all processes/SHOULD kill all processes/`

`s/and ensure no process left in it./and return an error if those processes can not be killed./`